mlrun/model_monitoring/api.py

```diff
@@ -704,8 +704,8 @@ def perform_drift_analysis(
         drift_detected_threshold=drift_threshold,
     )
 
-    # Drift table plot
-    html_plot = FeaturesDriftTablePlot().produce(
+    # Drift table artifact
+    plotly_artifact = FeaturesDriftTablePlot().produce(
```

Comment:
note that the `drift_results` here is from type `DriftResultType` that is defined within `batch.py` as 
`DriftResultType = tuple[mlrun.common.schemas.model_monitoring.DriftStatus, float]`

while you changed the corresponded param in `produce` function into a **different** `DriftResultType` that is defined within `features_drift_table.py` as 
`DriftResultType = tuple[
    mlrun.common.schemas.model_monitoring.constants.ResultStatusApp, float
]`

Reply:
Good catch.
The coming plan is to remove the `perform_drift_analysis` method altogether as a part of the batch removal, so it's in a temporary phase now.